packages/CalDAV-Core.package/CALSubscriptionICSCalendar.class/instance/caldavLink..st

```diff
@@ -1,3 +1,3 @@
 accessing
-caldavLink: anObject
-	caldavLink := anObject
\ No newline at end of file
+caldavLink: aLink 
```

Comment:
what is aLink? might be aString? => maybe check Transcript to find out class of argument